CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ filters in listing views [[GH-9442](https://github.com/hashicorp/consul/issues/9
 
 IMPROVEMENTS:
 
+* connect: Add new flags `prometheus-backend-port` and `prometheus-scrape-port`
```

Comment:
We don't edit the changelog directly. Instead we make `.changelog/<PR_NUMBER.txt>` files and the process of releasing will generate the changelog appropriately (it helps magically make backport changelog entries stay updated too)

Reply:
Thanks! Moved it to a .changelog